x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/index/IndexResolver.java

```diff
@@ -694,7 +695,7 @@ public void resolveAsSeparateMappings(
     }
 
     private static GetAliasesRequest createGetAliasesRequest(FieldCapabilitiesResponse response, boolean includeFrozen) {
-        return new GetAliasesRequest().aliases("*")
+        return new GetAliasesRequest(MasterNodeRequest.INFINITE_MASTER_NODE_TIMEOUT).aliases("*")
```

Comment:
This method is only used by the `sql` x-pack plugin, which I interpreted as internal use as well.